src/exchange/exchange.go

```diff
@@ -528,18 +565,26 @@ func (s *Exchange) broadcastTransaction(tx *coin.Transaction) (*sender.Broadcast
 	return rsp, nil
 }
 
-// BindAddress binds deposit btc address with skycoin address, and
-// add the btc address to scan service, when detect deposit coin
-// to the btc address, will send specific skycoin to the binded
+// BindAddress binds deposit btc/eth address with skycoin address, and
+// add the btc/eth address to scan service, when detect deposit coin
+// to the btc/eth address, will send specific skycoin to the binded
 // skycoin address
 // TODO -- support multiple coin types
-func (s *Exchange) BindAddress(skyAddr, btcAddr string) error {
+func (s *Exchange) BindAddress(skyAddr, btcAddr, coinType string) error {
+	//store to same bucket because btc address different with eth address
```

Comment:
Is this comment still valid? I thought you said you were using a different bucket

Reply:
many interface and unit tests will be changed if bind using different bucket, and next i will be try to implement that。
bind bucket will be bind_address_xxx, as follows
```
bind_address_btc
bind_address_eth
bind_address_xxx
``` 
and i will add coinType argument in most of function in exchange/store.go
is there a better solution ?

Reply:
deposit address has been used different bucket , and all unit test has been passed